src/snowflake/snowpark/open_telemetry_prototype.py

```diff
@@ -0,0 +1,75 @@
+import inspect
```

Comment:
Better make this file internal because you don't want to expose it to the users.